### PR TITLE
11/24 updates

### PR DESCRIPTION
**Pull request by aspiece**

### CODE REVIEW

File: docs/lab_51.md.html [hunk 1]
```diff
@@ -11,19 +11,22 @@
 
 ## Part 1: All together now
 
-Each of the six students in your group should follow these instructions.  Each student should then follow his or her instructions, completing each corresponding step together.  (For example, everybody should complete their step ii. at the same time.) Everyone should work on the same grid.
+1. Each of the six students in your group should follow these instructions.  Each student should then follow his or her instructions, completing each corresponding step together.  (For example, everybody should complete Part 2 at the same time. Everyone should work on the same grid.
```
Comment (Dan-TEALS) [inline docs/lab_51.md.html:14]: "his or her" is not inclusive. Better to use "their."

File: docs/lab_51.md.html [hunk 2]
```diff
@@ -11,19 +11,22 @@
...
+    5. Draw a line to the dot above.
+    6. Draw a line to the dot to the right.
+    7. Lift your pen.
 
-   ![Dots](images/dots.png)
+    ![Dots](images/dots.png)
 
-What was the result of the group following the instructions?  Were there any difficulties you encountered?  Does it seem like the result was intended?
+**Reflection**
+2. What was the result of the group following the instructions?
```
Comment (Dan-TEALS) [inline docs/lab_51.md.html:27]: What happened to 1?

File: docs/lab_51.md.html [hunk 3]
```diff
@@ -118,22 +121,20 @@
...
-| 1.2 Sprite controlled by stage                     | 0.5      |
-| 2.1 Clones created by stage                        | 0.5      |
-| 2.3 Hide master sprite                             | 0.5      |
-| 2.4 Sprites have different properties       | 0.5      |
-| **Total**                                  | **2.5** |
+| **Criteria** |Points |
+| :----- | :-- |
+| Part 1 participation and reflection completed | 0.5 |
+| Part 2 participation | 0.5 |
+| Part 3 refelction questions answered  | 0.5 |
```
Comment (Dan-TEALS) [inline docs/lab_51.md.html:135]: _reflection_ misspelled.

File: lab_51.md [hunk 4]
```diff
@@ -4,19 +4,22 @@ In this lab, you will follow sets of instructions to investigate how clones work
 
 ## Part 1: All together now
 
-Each of the six students in your group should follow these instructions.  Each student should then follow his or her instructions, completing each corresponding step together.  (For example, everybody should complete their step ii. at the same time.) Everyone should work on the same grid.
+1. Each of the six students in your group should follow these instructions.  Each student should then follow his or her instructions, completing each corresponding step together.  (For example, everybody should complete Part 2 at the same time. Everyone should work on the same grid.
```
Comment (Dan-TEALS) [inline lab_51.md:7]: "his or her" is not inclusive. Use "their."

Comment (Dan-TEALS) [inline lab_51.md:7]: Missing the close paren.

File: lab_51.md [hunk 5]
```diff
@@ -4,19 +4,22 @@ In this lab, you will follow sets of instructions to investigate how clones work
...
+    5. Draw a line to the dot above.
+    6. Draw a line to the dot to the right.
+    7. Lift your pen.
 
-   ![Dots](images/dots.png)
+    ![Dots](images/dots.png)
 
-What was the result of the group following the instructions?  Were there any difficulties you encountered?  Does it seem like the result was intended?
+**Reflection**
+2. What was the result of the group following the instructions?
```
Comment (Dan-TEALS) [inline lab_51.md:20]: What happened to 1?

File: lab_51.md [hunk 6]
```diff
@@ -84,15 +87,15 @@ What was the result of the group following the instructions?  Were there any dif
...
 1. What was the result of following the instructions?  
 2. Does this seem like the most efficient way to achieve this goal?  
 3. What would happen if the shape needed to be changed?  
 4. What problems could that cause?
 5. What might be a better or more efficient way to accomplish the goal of these instructions?
 
-## Part 3 - The same but different
+## Part 4 - The same but different
 
 Assign each member of your group a different number from 1 to 6.  Each of the six students in your group should then follow these instructions.  Each student should then follow his or her instructions, completing each corresponding step together.  (For example, everybody should complete their step ii. at the same time.) Everyone should work on the same grid.
```
Comment (Dan-TEALS) [inline lab_51.md:100]: Not one of the changes you made, but the instructions in this paragraph are also not inclusive. (Change "his or her" to "their.") Also, you changed the Roman numerals to a section number elsewhere. Should that have been done here?

File: lab_51.md [hunk 7]
```diff
@@ -111,20 +114,18 @@ Assign each member of your group a different number from 1 to 6.  Each of the si
...
-| 1.2 Sprite controlled by stage                     | 0.5      |
-| 2.1 Clones created by stage                        | 0.5      |
-| 2.3 Hide master sprite                             | 0.5      |
-| 2.4 Sprites have different properties       | 0.5      |
-| **Total**                                  | **2.5** |
+| **Criteria** |Points |
+| :----- | :-- |
+| Part 1 participation and reflection completed | 0.5 |
+| Part 2 participation | 0.5 |
+| Part 3 refelction questions answered  | 0.5 |
```
Comment (Dan-TEALS) [inline lab_51.md:128]: _reflection_ misspelled.

**Review by Dan-TEALS — CHANGES_REQUESTED** [review]
Some changes that should be made to the updated files before committing. I wasn't really able to review the slide deck changes.

File: docs/lab_51.md.html (hunk 2 above)
Comment (aspiece) [reply docs/lab_51.md.html:27]: Good question. LOL

**Comment by aspiece** [comment]
I am not sure where I used roman numerals. I try to stick to bullets and numbers. Andrew

**Comment by Dan-TEALS** [comment]
> 

On line 100 of the change file, there's a statement that "everybody should complete their step ii, at the same time." Elsewhere that was changed to "step 2."

**Comment by aspiece** [comment]
@nikmolnar  Thank you for the reminder. I needed to resolve the conflicts. The html version and ppt should be up to date now. I will update the lab documents in the next PR.

**Pull request merged**